### DISCUSSION TITLE
MacBook Air M1 > After Update Win11 > Black Screen

### DISCUSSION THREAD

**Discussion by cavemaster** [utmapp/UTM#6420] [Windows]
Hello everyone,

I have a MacMookAir and the latest versions of MacOSX and UTM.

Since the last Win11 update (Windows 11 Insider Preview 26231.5000 (ge_prerelease)), Win11 no longer starts after installation, the “screen” remains black after booting.
Win11 then resets itself automatically.

What can I do/check why the update does not work correctly with UTM?

Thank you!

**Comment by cavemaster** [comment]
No one has any ideas? :-(

**Reply by Kaiojag** [reply]
I have the same problem and have no idea what is happening. Have you solved it?

**Reply by cavemaster** [reply]
Unfortunately not.
I just reinstalled Win11.
